src/components/BigCardListItem.js

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
+import React, { memo } from 'react';
+import { Platform, StyleSheet, View } from 'react-native';
+import { Card, Divider } from 'react-native-elements';
+
+import { colors, consts, normalize } from '../config';
+import { imageHeight, imageWidth } from '../helpers';
+
+import { Image } from './Image';
+import { CategoryText, HeadlineText, RegularText } from './Text';
+import { Touchable } from './Touchable';
+
+export const BigCardListItem = memo(({ navigation, horizontal, item }) => {
```

Comment:
what does memo to in this part of the code?

Reply:
the `memo` in the code is a higher-order component from React that helps with performance optimization. It essentially memorizes the rendered output of the wrapped component and prevents unnecessary re-renders if the props haven't changed.

Reply:
thats cool, thank you for your explanation 😄